src/Npgsql/PostgresDatabaseInfo.cs

```diff
@@ -201,161 +203,135 @@ internal async Task<List<PostgresType>> LoadBackendTypes(NpgsqlConnector conn, N
             if (timeout.IsSet)
                 commandTimeout = (int)timeout.CheckAndGetTimeLeft().TotalSeconds;
 
-            using var batch = new NpgsqlBatch(conn)
-            {
-                BatchCommands =
-                {
-                    new("SELECT version()"),
-                    new(GenerateLoadTypesQuery(SupportsRangeTypes, conn.Settings.LoadTableComposites)),
-                    new(GenerateLoadCompositeTypesQuery(conn.Settings.LoadTableComposites)),
-                }
-            };
+            var batchQuery = new StringBuilder()
+                .AppendLine("SELECT version();")
```

Comment:
I'd put an extra blank line between the queries but up to you.

Reply:
You mean that you prefer **two** blank lines between the queries?
Mind you, the query strings returned by the generator methods start with a newline already.

Reply:
>  Mind you, the query strings returned by the generator methods start with a newline already.

Right, that's the part I missed - makes sense.